webpack/assets/javascripts/react_app/components/Editor/components/EditorSettings.js

```diff
@@ -116,7 +115,7 @@ const EditorSettings = ({
     <Button className="editor-button" id="cog-btn" bsStyle="link">
       <Icon size="lg" name="cog" />
     </Button>
-  </OverlayTrigger>
+  </Popover>
```

Comment:
there are a few leftovers from the old implementation like `placement`, `trigger`